### ISSUE TITLE
TS7016: Could not find a declaration file for module 'random-seed'

### ISSUE THREAD

**Issue by moltar** [Atrox/haikunatorjs#131]
```
❯ tsc
node_modules/haikunator/dist-types/index.d.ts:1:28 - error TS7016: Could not find a declaration file for module 'random-seed'. '~/node_modules/random-seed/index.js' implicitly has an 'any' type.
  Try `npm install @types/random-seed` if it exists or add a new declaration (.d.ts) file containing `declare module 'random-seed';`

1 import { RandomSeed } from 'random-seed';
                             ~~~~~~~~~~~~~


Found 1 error.
```

I think `@types/random-seed` needs to be installed as a dependency in `haikunator`?

Installing `@types/random-seed` locally in the project does fix the problem.

**Comment by Atrox** [comment]
https://github.com/Atrox/haikunatorjs/blob/dc8c4fce0370a43c321617c2a690e20d4293d312/package.json#L42

It is a dependency. Did you run `yarn install`?

**Comment by moltar** [comment]
It’s a dev dependency, which doesn’t get installed when installing from npm.

**Comment by Atrox** [comment]
Hey @moltar,
sorry for the oversight. This is fixed with v2.1.2
